src/main.cpp

```diff
@@ -509,8 +511,13 @@ class Konstructs: public nanogui::Screen {
     void handle_belt(const string &str) {
         uint32_t column, size, type, health;
         if(sscanf(str.c_str(), ",%u,%u,%u,%u",
-                  &column, &size, &type, &health) != 4)
-            throw std::runtime_error(str);
+            &column, &size, &type, &health)!=4)
+        {
+            std::string what="handle_belt expected ,%u,%u,%u,%u got "+str;
+
+            cout<<what;
+//            throw std::runtime_error(what);
+        }
```

Comment:
@caseymcc Server master/HEAD and client master/HEAD is not always in perfect sync (sorry about that, maybe we should keep them so). We are making release tags that are in sync like the latest release 6 https://github.com/konstructs/client/releases/tag/6 and https://github.com/konstructs/server/releases/tag/v0.1.17. Between them there are changes to the server <-> client protocol that are not always backwards compatible. This is one such change.